private-pools-example/README.md

```diff
@@ -0,0 +1,287 @@
+
+```
+ILB=10.4.0.5
+```
+
+Now that we have our private resources set up, we can now configure Cloud Build Private Pools. 
+
+## Configure your VPC for Cloud Build Private Pools
+
+Cloud Build Private Pools run workers using the Service Networking API. This enables you to offer your managed services on internal IP addresses. This is achieved by peering the Google-managed VPC running Cloud Build Private Pool workers with your own VPC.
+
+>Note: Cloud Build Private Pools only need to be configured via this method when builds need to access resources over a private network. If you are only looking to use Cloud Build for isolating builds to regions or for getting access to greater optionality of machine types, you do not need to do this. 
+
+
+We will ensure that the Service Networking API is enabled.
+```
+gcloud services enable servicenetworking.googleapis.com
+```
+
+Next, we will specify a named IP range for the Cloud Build Private Pool to utilize for workers. This must not overlap with any IP ranges we have in our VPC. You can either specify the range explicitly by providing `--address` and `--prefix-length`, or in the below example, you can allow GCP to provision an available range based on a provided `prefix-length`. 
+
+In this example, we are allocating a /24 range for our Cloud Build Private Pool to utilize for workers.
+
+```
+ gcloud compute addresses create $PRIVATE_POOLS_IP_RANGE_NAME \
+      --global \
+      --purpose=VPC_PEERING \
+      --prefix-length=24 \
+      --network=$VPC_NAME
+```
+Your output should look similar to the below.
+```
+Created [https://www.googleapis.com/compute/v1/projects/agmsb-k8s/global/addresses/sandbox-ip-range-private-pools].
+```
+
+Now, we can create a peering between the Google-managed VPC and our VPC running our private resources.
+
+```
+gcloud services vpc-peerings connect \
+    --service=servicenetworking.googleapis.com \
+    --ranges=$PRIVATE_POOLS_IP_RANGE_NAME \
+    --network=$VPC_NAME \
+    --project=$PROJECT_ID
+```
+
+Your output will look similar to the below.
+```
+Operation "operations/pssn.p24-381015962062-12a0a42b-3152-41d0-aaff-0ab4cdb1bde8" finished successfully.
+```
+
+## Creating a Cloud Build Private Pool 
+
+Now we can create our Private Pool. We will be editing the `privatepool.yaml` config file to configure the Private Pool settings.
+
+In line 2 of `privatepool.yaml`, replace `<add-project-number>` and `<add-vpc-name>` with your GCP Project number and the value stored in `$VPC_NAME` respectively. 
+
+```
+    peeredNetwork: projects/<add-project-number>/global/networks/<add-vpc-name>
+```
+
+If you prefer to change the machine type, disk size, or disable Public IPs on the Cloud Build Private Pool, you can do so in lines 4-6. 
+
+```
+gcloud alpha builds worker-pools create $PRIVATE_POOL_NAME --config-from-file privatepool.yaml --region $REGION
```

Comment:
use the GA version of this command: https://cloud.google.com/build/docs/private-pools/create-manage-private-pools

Reply:
Up to date.

Reply:
Updated in https://github.com/GoogleCloudPlatform/cloud-build-samples/pull/13/commits/97d3ee671c16fbf95a5ddbce21d0b41a466cf2ab